Solutions/ShadowByte Aria/Playbooks/ShadowByte_Aria_Enrich_Incidents/azuredeploy.json

```diff
@@ -8,15 +8,22 @@
         "prerequisites": [
             "1. ShadowByte Aria Custom Connector needs to be deployed and configured prior to the deployment of this playbook under the same resource group."
         ],
-        "prerequisitesDeployTemplateFile": "../ShadowByte_Custom_Connector/azuredeploy.json",
```

Comment:
@v-sabiraj : we should not delete any entry from playbook metadata object, please add prerequisitesDeployTemplateFile back